app/assets/stylesheets/styles/components/pages/_featured_crops.css.scss

```diff
@@ -0,0 +1,42 @@
+.featured-crops {
+  background-color: rgb(240, 245, 240);
+  padding: 40px inherit 20px;
+
+  .featured-crops-grid {
+    margin-top: 30px;
+
+    .featured-crop-container {
+      margin-bottom: 30px;
+
+      .featured-crop {
+        position: relative;
+        height: 280px;
+
+        &:hover {
+          .blackness {
+            opacity: 1;
+            @include single-transition(opacity, 200ms, ease-in-out);
+          }
+        }
+      }
+
+      .blackness {
+        background-color: rgba(0,0,0,0.45);
```

Comment:
`0.45` should be written without a leading zero as `.45`<br>Commas in function arguments should be followed by a single space